tests/Source/AbstractPropertySourceIteratorTest.php

```diff
@@ -13,36 +13,31 @@
 
 namespace Sonata\Exporter\Tests\Source;
 
-use Doctrine\ORM\Configuration;
-use Doctrine\ORM\EntityManager;
-use Doctrine\ORM\Query;
 use PHPUnit\Framework\TestCase;
-use Sonata\Exporter\Tests\Source\Fixtures\DoctrineORMQuerySourceIterator;
+use Sonata\Exporter\Source\AbstractPropertySourceIterator;
 use Sonata\Exporter\Tests\Source\Fixtures\ObjectWithToString;
 
-/**
- * @author Joseph Maarek <josephmaarek@gmail.com>
- */
-final class DoctrineORMQuerySourceIteratorTest extends TestCase
+final class PropertySourceIterator extends AbstractPropertySourceIterator
 {
-    /**
-     * @var Query
-     */
-    private $query;
+    public function rewind(): void
+    {
+        $this->iterator->rewind();
+    }
 
-    protected function setUp(): void
+    public function getValue($value)
     {
-        $entityManager = $this->createMock(EntityManager::class);
-        $entityManager->method('getConfiguration')->willReturn(new Configuration());
-        $this->query = new Query($entityManager);
+        return parent::getValue($value);
     }
+}
 
+final class AbstractPropertySourceIteratorTest extends TestCase
```

Comment:
Is there a reason why you define two classes in this file?

Reply:
Have you considered using an anonymous class for that purpose?

Reply:
I never used an anonymous class. But I don't think it would be better with it. Maybe is it because I'm not used to.

Reply:
I like to use them for that kind of thing partly because it nips these sort of discussions about several classes per file in the bud but as you wish, I'm fine with both.

Reply:
I'm fine with everything. I can change if @sonata-project/contributors prefer the anonymous class, but a gist with the anonymous class could help me a lot.